src/Secciones/BarraCircular.java

```diff
@@ -15,7 +15,7 @@ public void area(){
     }
 
     public void Baricentro(){
-        System.out.println("Baricentro: al ser doblemente simetrica este se encuentra en el centro de la seccion");
+        System.out.println("Baricentro: al tener simetria polar, este se encuentra en el centro de la seccion");
```

Comment:
Por ahí no entiendo el contexto, pero si el baricentro se encuentra en el centro de la sección, no me podrías devolver las coordenadas donde se encuentra? Ejemplo, podes devolver un objecto que se llame Coordenada (o algo por el estilo) que tenga de atributos coordenadaEnX y coordenadaEnY. Digo porque eso sería mucho más útil para el que lo usa, que usar en cambio un System.out.print()


`public class Coordenada {
    private final int coordenadaEnX;
    private final int coordenadaEnY;

    public Coordenada(int coordenadaEnX, int coordenadaEnY) {
        this.coordenadaEnX = coordenadaEnX;
        this.coordenadaEnY = coordenadaEnY;
    }

    public int getCoordenadaEnX() {
        return coordenadaEnX;
    }

    public int getCoordenadaEnY() {
        return coordenadaEnY;
    }
}`